test/SparqlAntlrParserTest.cpp

```diff
@@ -598,6 +598,29 @@ TEST(SparqlParser, VarOrTermGraphTerm) {
   EXPECT_THAT(varOrTerm, IsIri(nil));
 }
 
+TEST(SparqlParser, Iri) {
+  auto expectIri = [](const string& input, string iri,
+                      SparqlQleverVisitor::PrefixMap prefixMap = {}) {
+    // Parse "by hand" in order not to pollute the `SparqlParserHelpers`.
```

Comment:
In one of the other PRs I have suggested getting rid of these helpers at all (which seems feasible now). You could add a TODO here to not forget this place once we have this change.